docs/Deploying/Configuration.md

```diff
@@ -2944,6 +2946,70 @@ For this to work correctly make sure that:
   - `urn:oid:2.5.4.3`
   - `http://schemas.xmlsoap.org/ws/2005/05/identity/claims/name`
 
+#### Customization of user roles
+
+Users created via JIT provisioning can be set Fleet roles using SAML custom attributes that are sent by the IdP in `SAMLResponse`s during login.
```

Comment:
Small typo:
` can be set Fleet roles` => `can be assigned Fleet roles`

Reply:
Thanks! Fixed c83794bc6.